README.md

```diff
@@ -3,7 +3,7 @@
 
 Avatar Animator is a tool for generating real-time 2D vector-based avatar animations using webcams or other video devices. It can be used with Zoom, Skype, Microsoft Team, Google Meet, and many more video-conferencing apps. The project is a fork of [Pose Animator](https://github.com/yemount/pose-animator), a fast and real-time 2D vector illustrations animator based on Google's [PoseNet](https://github.com/tensorflow/tfjs-models/tree/master/posenet) and [FaceMesh](https://github.com/tensorflow/tfjs-models/tree/master/facemesh) model. It wraps Pose Animator in an Electron app and sends it to a virtual camera.
 
-![Gif animation showing a demo of Avatar Animator using Microsoft Teams](resources/gifs/demo.gif)
+![Gif animation showing a demo of Avatar Animator using Microsoft Teams](assets/demo.gif)
```

Comment:
I don't think assets is any better. Normally assets is used for things that become part of the app/game. Maybe just images/?

Reply:
Sure -- done in https://github.com/letmaik/avatar-animator/pull/23/commits/4fb15c652433b74e6161b55b2e899e773d780d35